### ISSUE TITLE
bug: initialized with all options selected, but "check all" checkbox not checked

### ISSUE THREAD

**Issue by jonirrings** [sa-si-dev/virtual-select#92]
to reproduce
```html
<div id="sample-select"></div>
```
```js
var el = document.getElementById('sample-select');
var opts = [
        {label: 'Options 1', value: '1'},
        {label: 'Options 2', value: '2'},
        {label: 'Options 3', value: '3'},
    ];
var initialValue = opts.map(opt=>opt.value);

VirtualSelect.init({
    ele: el,
    multiple: true,
    enableSecureText: true,
    options: opts,
    selectedValue: initialValue,
});
```
and click the `select` component, after the dropbox expanded, you will see an unchecked `checkbox` aside the `Search` text.
with the options checkbox all checked.

**Comment by sa-si-dev** [comment]
@jonirrings thanks for letting me know it. I will fix it in the future release.

**Comment by sa-si-dev** [comment]
@jonirrings this issue has been fixed and available in the [release v1.0.24](https://github.com/sa-si-dev/virtual-select/releases/tag/v1.0.24)

**Comment by jonirrings** [comment]
@sa-si-dev nice and thanks for your awesome work.
